app/models/benefit_receipt.rb

```diff
@@ -1,5 +1,8 @@
 class BenefitReceipt < ApplicationRecord
   extend EnumHash
+  extend PaymentPatternConcern
+  define_payment_pattern
```

Comment:
Not too sure about this method name. `has_payment_pattern` seemed a better fit, but rubocop complained.